lib/services/ngsi/entities-NGSI-v2.js

```diff
@@ -970,16 +970,22 @@ function sendUpdateValueNgsi2(entityName, attributes, typeInformation, token, ca
         // only in the first case
         if (options.json.entities.length === 1) {
             // recreate options object to use single entity update
-            url = '/v2/entities/' + entityName + '/attrs';
-            if (typeInformation && typeInformation.type) {
-                url += '?type=' + typeInformation.type;
+            url = '/v2/entities';
+            if (config.getConfig().appendMode === false) {
+                url += '/' + entityName + '/attrs';
+                if (typeInformation && typeInformation.type) {
+                    url += +'?type=' + typeInformation.type;
```

Comment:
I think this should be:

```suggestion
                    url += options.json.id +'/attrs?type=' + typeInformation.type;
```

(Not sure if `options.json.id` is the right variable here to get entity id)